src/backend/app.js

```diff
@@ -4,7 +4,8 @@ import mealsRouter from "./api/meals.js";
 import cors from "cors";
 import { fileURLToPath } from 'url';
 import { dirname } from 'path';
-import knex from 'knex';
+//import knex from 'knex';
```

Comment:
This change seems unintentional and keeps me from running the server. 🥺 It should probably just be removed altogether.

Reply:
I had made a mess with branches🥲 but now fixed this